_themes/sass/_theme_type.sass

```diff
@@ -0,0 +1,236 @@
+// Typography
```

Comment:
Allows me to custom style the typography instead of having it be part of a bower dependency.